lib/winSubset.ts

```diff
@@ -0,0 +1,398 @@
+/**
+ * List of exported symbole names from libclang.dll
+ * obtained with `choco install --version 14.0.6 llvm`
+ * md5 59beb52cef40898b0f24cdffc6cf2984
+ * `dumpbin /exports libclang.dll`
+ */
+export const winSubset = [
```

Comment:
Constants must be in ALL_UPPER_CASE_WITH_UNDERSCORES. Please also use the same name that Deno uses for the architecture, so `WINDOWS_SUBSET`.

Also: Please make this into a negative set, ie. those keys that are missing. While I do understand that this is easier to generate, I would rather not duplicate the names of nearly every symbol in the libclang API here.

Reply:
Mmm, interesting point. I will do it that night.

Reply:
I'm finally not shure that the way to go.
I can include a subset of 306 function, or exclude a subset of 920 function something like that.

A proper way to process is to have a 2 const IMPORTS.
So it is possible to have a proper IntelliSense for calls that is available cross-platform.

Reply:
Huh, I figured the list of unincluded symbols would be a select few. If roughly three out of every four symbols are not present in the Windows DLL then... Well, then at least your original plan of having a Set is definitely better than my plan of an ignore array.

But... It does not sound correct that there would be so many APIs missing from the Windows DLL.

Reply:
I have implement the strict version now the code looks like:

```ts
export type WindowsSubSet = typeof WINDOWS_SUBSET[number]
type clangExpUnix = typeof IMPORTS
type clangExpCommon = Pick<clangExpUnix, WindowsSubSet>

let libclang: ReturnType<typeof Deno.dlopen<clangExpUnix | clangExpCommon>>;
```
I rename `windows` function to `common` function.

Reply:
by the way, 920 is based on the number of the exported symbols from the so file.
I think your generator did not output most of them. SO I still have to count them... and double check that

Reply:
after some more check:

I gonna follow your original idea :smiling_face_with_tear: